src/api/layers/wmts-capabilities.class.js

```diff
@@ -0,0 +1,138 @@
+import { LayerAttribution } from '@/api/layers/AbstractLayer.class'
+import ExternalWMTSLayer from '@/api/layers/ExternalWMTSLayer.class'
+import OlWmtsCapabilities from 'ol/format/WMTSCapabilities'
+import log from '@/utils/logging'
+
+/** Wrapper around the OpenLayer WMSCapabilities to add more functionalities */
+export default class WmtsCapabilities {
```

Comment:
same remark as with `WmsCapabilites` as only the lower cased `ms` are different from the OL counterpart, we could easily confuse them when importing

Reply:
Done